Tests/Functional/Domain/Repository/TeaRepositoryTest.php

```diff
@@ -31,6 +32,14 @@ protected function setUp(): void
         $this->subject = $this->get(TeaRepository::class);
     }
 
+    /**
+     * @test
+     */
+    public function isRepository(): void
```

Comment:
I don't see a reason to add this method. Can you explain?

Reply:
The rationale for the existence of this test (not for moving it) is to test that our repository extends the base class and hence can be trusted to have all the behavior of the base class (which hopefully is test-covered in Extbase). Otherwise, we'd need to have tests for the base class methods to cover the full code of the class.